cognite/powerops/preprocessor/main.py

```diff
@@ -27,9 +27,7 @@ def main(
     cog_shop_case = cog_reader.cog_shop_case.to_dict()
     cog_shop_case_yaml = yaml.dump(cog_shop_case, allow_unicode=True, encoding="utf-8")
 
-    extra_files_md = cog_reader.get_extra_files_metadata()
```

Comment:
When you agree with the logic, should we remove these methods or keep them in the case where cogshop file config does not exist in cdf yet?

Reply:
Yes, I think you should have backwards compatibility. So yes maybe an else clause to handle if there  is no cogshop file list.